openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -115,6 +117,52 @@
             return deferred.promise();
         },
 
+        loadStaffGradeForm: function(eventObject) {
+            var view = this;
+            var $staffGradeTab = $('.openassessment__staff-grading', this.element);
+            var isCollapsed = $(eventObject.currentTarget).hasClass("is--collapsed");
+            var deferred = $.Deferred();
+            var showFormError = function(errorMessage) {
+                $staffGradeTab.find('.staff__grade__form--error').text(errorMessage);
+            };
+
+            if (isCollapsed && !this.staffGradeFormLoaded) {
```

Comment:
We still want to work out the expected behavior when the grading section is collapsed or we move to another tab.